packages/ui/src/pages/SignIn/index.module.scss

```diff
@@ -12,12 +11,12 @@
   position: relative;
   padding: _.unit(8);
   height: 100%;
-  @include flex-colomn;
+  @include flex-column;
 }
 
 .form {
   width: 100%;
-  @include flex-colomn;
+  @include flex-column;
```

Comment:
Should include `flex-column` mixin from `underscore`

Reply:
updated